saleor/payment/__init__.py

```diff
@@ -1,19 +1,20 @@
-import importlib
 from enum import Enum
 
 from django.conf import settings
-from django.core.exceptions import ImproperlyConfigured
 from django.utils.translation import pgettext_lazy
 
-from .interface import GatewayConfig
-
 
 class PaymentError(Exception):
     def __init__(self, message):
         super(PaymentError, self).__init__(message)
         self.message = message
 
 
+GATEWAYS_ENUM = Enum(
+    "GatewaysEnum", {key.upper(): key.lower() for key in settings.PAYMENT_GATEWAYS}
```

Comment:
@salwator Maybe we could iterate over a list of payment plugins to get their names and generate this ENUM based on this? Thanks to it we will not need `PAYMENT_GATEWAYS` anymore.

Reply:
We can't easily do that right now and I don't really think it is ultimately the way to go, we might want to enable plugins in runtime in the future. I don't think API spec should depend on plugin discovery process 🤷‍♂️